code/src/index.css

```diff
@@ -1,13 +1,183 @@
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+    background-color: #191414;  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* styling for main section */
+.main-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+/* styling for header */
+.header-container {
+  margin: 0px 20px;
+  grid-column: span 4;
+}
+
+.header-text {
+  color: #fff;
+  font-size: 24px;
+  border-bottom: 1px solid rgb(55, 55, 55);
+  padding-bottom: 5px;
+  text-align: left;
+}
+
+/* styling for each album card */
```

Comment:
I like the way you add comments for each section of styling. Makes it easy to follow the code!